modules/imaging_browser/locale/imaging_browser.pot

```diff
@@ -29,3 +29,87 @@ msgstr ""
 
 msgid "New and pending imaging sessions"
 msgstr ""
+
+msgid "Visit QC Status"
+msgstr ""
+
+msgid "First Acquisition"
+msgstr ""
+
+msgid "First Insertion"
+msgstr ""
+
+msgid "Last QC"
+msgstr ""
+
+msgid "New Data"
+msgstr ""
+
+msgid "Links"
+msgstr ""
+
+msgid "SessionID"
+msgstr ""
+
+msgid "Sequence Type"
+msgstr ""
+
+msgid "Pending New"
+msgstr ""
+
+msgid "Entity Type"
+msgstr ""
+
+msgid "QC Status"
+msgstr ""
+
+msgid "selected"
+msgstr ""
+
+msgid "all types"
+msgstr ""
+
+msgid "NEW"
+msgstr ""
+
+msgid "Pass"
+msgstr ""
+
+msgid "Fail"
+msgstr ""
+
+msgid "Header Info"
+msgstr ""
+
+msgid "Selected"
```

Comment:
Can you also make sure "Selected" (uppercase) is next to "selected" (lowercase) in the `.pot` file to make it easier to translate?